addons/binding/org.openhab.binding.powermax/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Powermax Binding
+Bundle-SymbolicName: org.openhab.binding.powermax;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: com.google.common.collect,
+ gnu.io,
+ javax.xml.bind,
```

Comment:
this isn't part of CP2 and you only seem to need it for `printHexBinary`. Could you use `org.eclipse.smarthome.core.util.HexUtils` instead?

Reply:
I changed in favor of HexUtils.bytesToHex.